phpseclib/File/X509.php

```diff
@@ -3117,12 +3118,13 @@ public function removeExtension($id)
      *
      * @param string $id
      * @param array $cert optional
+     * @param string $path
      * @access public
      * @return mixed
      */
-    public function getExtension($id, $cert = null)
+    public function getExtension($id, $cert = null, $path)
```

Comment:
Never mind this - it looks like you fixed this specific issue with https://github.com/phpseclib/phpseclib/pull/1157/commits/eb21fb2bcb781a25c718cdf19f24bc59ea83ae2d